packages/runtime/id-compressor/src/test/testCommon.ts

```diff
@@ -16,7 +16,6 @@ import {
 /**
  * An identifier (v4 UUID) that has been shortened by a distributed compression algorithm.
  * Lacks a space (session/op), meaning its scope is the same as the space-specific ID from which it was derived.
- * @alpha
```

Comment:
Was this intentional?

Reply:
Yes - that is internal test code - listing in description as step 2

Reply:
Now broken out to #21446
It is not a strict requirement - just good not to have to avoid the search and replace hitting it